package.json

```diff
@@ -26,17 +26,25 @@
     "webpack": "^5.53.0",
```

Comment:
I think we can leave these as regular dependencies. 

1. glob
2. js-yaml 
3. path-complete-extname

For sure, anything with the name webpack or babel in it should be a peer.